### PR TITLE
Be app 0103

### PR DESCRIPTION
**Pull request by ralsalloum**
- fixtures bundle was added.
- package category and package fixtures were added.
- storeOwnerProfileUpdate function updated.

### CODE REVIEW

**Review by Kenan-Hussein — CHANGES_REQUESTED** [review]
Request changes

**Review by Kenan-Hussein — CHANGES_REQUESTED** [review]
changes requested

File: backend/src/Controller/Commander/DataCommanderController.php [hunk 1]
```diff
@@ -50,29 +52,37 @@ public function appendNewData(KernelInterface $kernel): Response
...
-//        return new Response($content);
-//    }
+    /**
+     * @Route("reenternewdata", name="reenterNewData", methods={"POST"})
+     * @IsGranted("ROLE_SUPER_ADMIN")
+     * @param KernelInterface $kernel
+     * @return Response
+     * @throws Exception
+     */
+    public function createNewData(KernelInterface $kernel): Response
```
Comment (Kenan-Hussein) [inline backend/src/Controller/Commander/DataCommanderController.php:70]: this is good, but we do not need it at all, 
after drop DB we can call just service for fill DB again.

Comment (Kenan-Hussein) [reply backend/src/Controller/Commander/DataCommanderController.php:70]: meaning we do not need controller or api for it

File: backend/fixtures/003_PackageCategory.yml [hunk 2]
```diff
@@ -5,8 +5,12 @@ App\Entity\PackageCategoryEntity:
 
   second_category:
     name: الباقات المتزامنة B
-    description: تتيح لك الطلب طول فترة دوام المؤسسة/الشركة. عدد 2 كباتن في الوقت الواحد كحد أقصى. مسافة 50 كيلومتر كحد أقصى من موقع المحل (استلام الطلب). عند دفع 99 ريال فقط ستتمكن من إضافة سائق ثالث للباقة لمدة شهر.
+    description: تتيح لك الطلب طول فترة دوام المؤسسة/الشركة. عدد 2 كباتن في الوقت الواحد كحد أقصى. مسافة 50 كيلومتر كحد أقصى من موقع المحل (استلام الطلب). عند دفع 99 ريال فقط ستتمكن من إضافة سائق رابع للباقة لمدة شهر.
```
Comment (Kenan-Hussein) [inline backend/fixtures/003_PackageCategory.yml]: please contact @rahafff and change the data content to be something better

**Review by Kenan-Hussein — CHANGES_REQUESTED** [review]
Request changes

File: backend/src/Controller/Commander/DataCommanderController.php (hunk 1 above)
Comment (ralsalloum) [reply backend/src/Controller/Commander/DataCommanderController.php:70]: If we need to use it in the system control app, we need an API, right?

Comment (Kenan-Hussein) [reply backend/src/Controller/Commander/DataCommanderController.php:70]: no, as I said in my first comment in second line

**Review by Kenan-Hussein — APPROVED** [review]

**Pull request merged**